generators/app/templates/test/index.js

```diff
@@ -18,7 +18,7 @@ describe("<%= description %>", () => {
 
     it(`should ${caseName.split("-").join(" ")}`, () => {
       const actual = babel.transformFileSync(actualFile, {
-        plugins: [plugin]
+        plugins: [plugin(babel)]
       }).code;
```

Comment:
Not necessarily, see: https://github.com/gaearon/babel-plugin-react-transform/tree/master/test/fixtures

Options are being used more and more in Babel plugins, we should be encouraging testing of every option.